config/StrategyResolver.py

```diff
@@ -156,69 +156,47 @@ def confirm_harvest(self, before, after, tx):
             "bveCVX", "badgerTree"
         )
 
-        ## bveCVX is sent to Governance, just check balanceIncresed
-        assert after.balances("bveCVX", "governanceRewards") > before.balances(
-            "bveCVX", "governanceRewards"
+        ## CVX is sent to Governance, just check balanceIncresed
+        assert after.balances("cvx", "governanceRewards") > before.balances(
+            "cvx", "governanceRewards"
         )
 
         ## Strategist Perf fee is set to 0, no funds have moved
-        assert after.balances("bveCVX", "strategist") == before.balances(
-            "bveCVX", "strategist"
+        assert after.balances("cvx", "strategist") == before.balances(
+            "cvx", "strategist"
         )
 
-        # 80% of cvxCrvHelperVault shares were distributed through the tree
+        # 100% of cvxCrvHelperVault shares were distributed through the tree
         actual_cvx_crv_helper_tree = (
             (
                 after.balances("cvxCrv", "cvxCrvHelperVault")
                 - before.balances("cvxCrv", "cvxCrvHelperVault")
             )
             * (cvxCrvHelperVault.getPricePerFullShare() / 1e18)
-            * 0.8
         )
         actual_tree_cvx_crv_delta = after.balances(
             "cvxCrv", "badgerTree"
         ) - before.balances("cvxCrv", "badgerTree")
 
         assert actual_cvx_crv_helper_tree > actual_tree_cvx_crv_delta
```

Comment:
Possibly unrelated, but not sure what this assert is trying to compare

Reply:
It basically checks that the whole amount of bcvxCRV generated by the strat is distributed to the tree as it should.

Reply:
You are very right, completely missed that, it must be still passing because the delta is 0. Will fix it.

Reply:
Alright, I pushed for this. Thanks for the heads up!